projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -595,4 +595,10 @@ export interface OccEndpoints {
    * @member {string}
    */
   b2bUserPermission?: string | OccEndpoint;
+  /**
+   * Endpoint for schedule a replenishment order
```

Comment:
"Endpoint for schedule a replenishment order" -> "Endpoint to schedule a replenishment order"?